apps/src/componentLibrary/tags/tags.module.scss

```diff
@@ -0,0 +1,113 @@
+@import "color.scss";
+@import "font.scss";
+@import "@cdo/apps/componentLibrary/typography/typography.module";
+
+// Tags common styles
+.tags {
+  display: flex;
+  column-gap: 0.5em;
+
+  > div {
+    display: flex;
+    align-items: baseline;
+    gap: 2px;
+    color: $neutral-dark;
+    padding: 2px 12px;
+    background-color: $brand_primary_light;
+    border-radius: 6.25rem;;
+    max-width: 15em;
+
+    span {
+      white-space: nowrap;
+      overflow: hidden;
+      text-overflow: ellipsis;
+    }
+
+    i {
+      display: flex;
+      justify-content: center;
+    }
+
+    &:has(i:first-child) {
+      padding-inline-start: 8px;
+    }
+
+    &:has(i:not(:first-child)) {
+      padding-inline-end: 8px;
+    }
+  }
+}
+
+.tagTooltip {
+  opacity: 1;
+  margin-bottom: 0.75em;
+  pointer-events: none;
+
+  :global(.tooltip-inner) { /* stylelint-disable-line selector-pseudo-class-no-unknown */
+    border-radius: 5px;
+    color: #fff;
+    background-color: #222;
+
+    p {
+      color: #fff;
+      margin: 0;
+    }
+  }
+
+  :global(.tooltip-arrow) { /* stylelint-disable-line selector-pseudo-class-no-unknown */
+    left: 50%;
+    bottom: 0;
+    border: 15px solid transparent;
+    border-top: 15px solid #222;
+    transform: translate(-50%, calc(100% - 8px));
+  }
+}
+
+// Sizes
+.tags-l {
+  > div {
+    span {
```

Comment:
Should we classnames instead of element tags? Or is this needed to override some Bootstrap styles?

Reply:
Oh, that's just how I structured the styles initially. I think that's a good idea to use a className instead of `> div `, thanks for spotting this!
Though, if we're talking about also setting classNames to `span` and `i` inside of that `div` - I don't think that's the best way since it'll make styles and code harder to read IMO